firmware/src/network_model.c

```diff
@@ -333,14 +290,11 @@ int NetworkModel_SetDHCPMode(const RDMHeader *header,
 
 int NetworkModel_GetZeroconfMode(const RDMHeader *header,
                                  const uint8_t *param_data) {
-  unsigned int index = 0;
```

Comment:
Do you mean you don't like:
unsigned int index = 0;
int r = CheckInterfaceRequest(header, param_data, &index);
if (r >= 0) {
  return r;
}